### ISSUE TITLE
[Feature]  Air quality index:  Provide information on air quality and potential health risks.

### ISSUE THREAD

**Issue by Chandravan** [Mohit5Upadhyay/WeatherApplication#58]
Provide information on air quality and potential health risks.

**Comment by Chandravan** [comment]
I'd like to take on the development of this feature!
I'll implement the **Air Quality Index** functionality, which will provide real-time information on the air quality and potential health risks associated with different levels.
 Please assign me this task. Thank you!

**Comment by Mohit5Upadhyay** [comment]
Yes , @Chandravan go ahead to implement the real time AQI prediction related to current weather information and some suggestion related to health risk

**Comment by Chandravan** [comment]
Please assign the labels `gssoc-ext` and `hacktoberfest` to this issue. Thank you!
